### PR TITLE
feat: implement two slash in docs

### PR DESCRIPTION
**Pull request by alessey**
**What changed? Why?**
Added twoslash to all codeblocks

image

**Notes to reviewers**
Twoslash needs to be [explicitly triggered](https://github.com/wevm/vocs/blob/main/src/vite/plugins/mdx.ts#L107) on each codeblock

**How has it been tested?**

### CODE REVIEW

File: site/docs/pages/guides/lifecycle-status.mdx [hunk 1]
```diff
@@ -31,10 +31,13 @@ can tap into the rhythm and customize the behavior of your components. 🎶
...
+```tsx twoslash
+import { useCallback } from 'react';
+import { Transaction } from '@coinbase/onchainkit/transaction';
+// ---cut-before---
 import type { LifeCycleStatus } from '@coinbase/onchainkit/transaction';
 
-const handleOnStatus = useCallback(status: LifeCycleStatus) => {
+const handleOnStatus = useCallback((status: LifeCycleStatus) => {
   console.log('LifecycleStatus', status);
 }, []);
```
Comment (alessey) [inline site/docs/pages/guides/lifecycle-status.mdx:42]: TODO: update case when new version

File: site/docs/styles.css [hunk 2]
```diff
@@ -38,7 +40,7 @@
 
 .vocs_CodeGroup {
   border-radius: 8px;
-  overflow: hidden;
+  overflow: visible;
```
Comment (alessey) [inline site/docs/styles.css:43]: allow twoslash popups to overflow code boxes

Comment (Zizzamia) [reply site/docs/styles.css:43]: niiice

Comment (Zizzamia) [reply site/docs/styles.css:43]: Hopefullly this will not cause a regression with Mobile Web, but if it does we need to re-think this.

**Review by Zizzamia — APPROVED** [review]
Niiiice!

Great job

**Pull request merged**